apps/testingaccessibility/src/pages/api/auth/[...nextauth].ts

```diff
@@ -90,37 +124,30 @@ export const nextAuthOptions: NextAuthOptions = {
   callbacks: {
     async session({session, token}) {
       if (token?.id) {
-        const {getPurchasesForUser} = getSdk()
-        const user = await prisma.user.findUnique({
-          where: {id: token.id as string},
-        })
-        session.roles = user?.roles || []
-        const purchases = await getPurchasesForUser(token.id as string)
-        token.purchases = purchases
-        token.roles = user?.roles || []
-        session.rules = defineRulesForPurchases(purchases)
+        const user = await getUser(token.id as string)
+        if (user) {
+          const {roles, purchases} = user
+          session.purchases = purchases
+          session.role = roles
+          token.purchases = purchases
+          token.role = roles
+        }
       } else {
         token.purchases = []
+        session.purchases = []
       }
 
-      const encodedToken = jwt.sign(token, process.env.NEXTAUTH_SECRET || '', {
-        algorithm: 'HS256',
-      })
-
-      // this gives us access to the token in the browser via getSession()
-      session.id = token.id
-      session.token = encodedToken
-
       return session
     },
-    async jwt({token, profile, account, user}) {
-      if (user) {
-        token.id = user.id
-        const {getPurchasesForUser} = getSdk()
-        const purchases = await getPurchasesForUser(user.id)
-        token.purchases = purchases
-        token.roles = user?.roles || []
-        token.rules = defineRulesForPurchases(purchases)
```

Comment:
this callback was a mess so cleaning it up and making it more consistent for use with rules engine